website/styles/swiper.css

```diff
@@ -45,3 +47,49 @@
 .food-category-swiper .swiper-button-next:after {
   font-size: 20px;
 }
+
+@media (max-width: 575px) {
```

Comment:
Let's try to use standard breakpoints if feasible.

Reply:
we have adjust media screen size on our requirement that's why used it.
![image](https://github.com/canopas/bite-space/assets/123158106/1a17d297-2a5a-48b6-a563-ef0830440506)